app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
```

Comment:
If you have 1 element after the bracket and then transfer to the next line, then in this case this element should also be on a new line